lib/icinga/checkable.hpp

```diff
@@ -237,6 +236,9 @@ class I2_ICINGA_API Checkable : public ObjectImpl<Checkable>
 	std::set<intrusive_ptr<Dependency> > m_ReverseDependencies;
 
 	void GetAllChildrenInternal(std::set<Checkable::Ptr>& children, int level = 0) const;
+
+	/* Flapping */
```

Comment:
Was this moved intentionally?

Reply:
Yep. Flapping calculation should only be triggered by a check result update, but not called from anywhere else.